src/components/Checkout/OrderTotals.vue

```diff
@@ -276,44 +254,18 @@ export default {
 				this.setUpdating(false);
 			});
 		},
-		removeCredit(type) {
```

Comment:
@ryan-ludwig We need to add this method back in so that standard credit removal continues to function.